content/en/_index.html

```diff
@@ -102,83 +94,76 @@ <h5 class="card-title text-white section-head">Model serving</h5>
         />
         <div class="card-body bg-primary-dark">
-          <h5 class="card-title text-white section-head">Multi-framework</h5>
+          <h5 class="card-title text-white section-head">Model Training</h5>
           <p class="card-text text-white">
-            Our development plans extend beyond TensorFlow. We're working
-			hard to extend the support of 
-			<a target="_blank" rel="noopener" href="/docs/components/training/pytorch/">PyTorch</a>, 
-			<a target="_blank" rel="noopener" href="/docs/components/training/mxnet/"> Apache MXNet</a>, 
-		  	<a target="_blank" rel="noopener" href="/docs/components/training/mpi/"> MPI</a>,
-		  	<a target="_blank" rel="noopener" href="/docs/components/training/xgboost/"> XGBoost</a>,
-			<a target="_blank" rel="noopener" href="/docs/components/training/chainer/"> Chainer</a>, and more. We
-			also integrate with 
-			<a target="_blank" rel="noopener" href="https://istio.io/">Istio</a> and 
-			<a target="_blank" rel="noopener" href="https://www.getambassador.io/">Ambassador</a> for ingress, 
-			<a target="_blank" rel="noopener" href="https://nuclio.io/">Nuclio</a> as a
-            fast multi-purpose serverless framework, and <a target="_blank" rel="noopener" href="https://www.pachyderm.io/">Pachyderm</a> for managing
-            your data science pipelines.
+            <a href="/docs/components/training/" target="_blank" rel="noopener" >Kubeflow Training Operators</a>
+            provide a unified, Kubernetes interface for training machine learning models on multiple frameworks including PyTorch, TensorFlow, MPI, MXNet, PaddlePaddle, and XGBoost.
           </p>
         </div>
       </div>
       <div class="card border-primary-dark">
         <img
           src="/docs/images/logos/Community.png"
-		  class="card-img-top"
-		  draggable="false"
+          class="card-img-top"
+          draggable="false"
           style="padding: 2rem;"
           alt="People Icon"
         />
         <div class=" card-body bg-primary-dark">
           <h5 class="card-title text-white section-head">Community</h5>
           <p class="card-text text-white">
-            We are an open and welcoming community of software developers, data scientists, and organizations!
-            Join our <a target="_blank" rel="noopener" href="/docs/about/community/#kubeflow-slack">Slack Workspace</a>!
+            We are an open and welcoming <a href="/docs/about/community/" target="_blank" rel="noopener">community</a> of software developers, data scientists, and organizations!
           </p>
         </div>
       </div>
     </div>
     <br />
-    <!-- For future editors: If you are adding cards not in multiples of 3, wrap them in a 'card-columns' div 
-						instead of card-deck. It will look more pleasing to the eye. -->
+    <!-- For future editors: If you are adding cards not in multiples of 3, wrap them in a 'card-columns' div instead of card-deck.
+         It will look more pleasing to the eye. -->
```

Comment:
What about Kubeflow Central Dashboard, Profile Controller, Volumes UI, Tensorboards ? Should we set a section for those components ?
 cc @kubeflow/wg-notebooks-leads

Reply:
@andreyvelich The only one I think might deserve a call out on the homepage is the central dashboard + multi-user support, because it highlights that Kubeflow is a platform, not just a collection of tools.

Perhaps the card could be:

__Title:__ "Central Dashboard"

__Picture:__  _(an outline of a house/home in a blue heptagon)_

__Description:__ 
"The Central Dashboard provides an authenticated web interface for Kubeflow components and ecosystem add-ons. It acts as a hub for your machine learning platform and tools."

Reply:
I have also just spent the day rewriting the central dashboard docs so they are up to date and not misleading if we link to them on the front page, see PR: https://github.com/kubeflow/website/pull/3635

Reply:
Sounds good, would like to know @kimwnasptd thoughts on it.

Reply:
I have replaced the community card with "dashboard" and created a new logo for it, and now that https://github.com/kubeflow/website/pull/3635 was merged, its actually a good place to link users.